### PR TITLE
feat(product): image gallery

### PR DESCRIPTION
**Pull request by emersonlaurentino**
## What's the purpose of this pull request?
Image gallery in desktop and mobile

## How it works? 
<em>Tell us the role of the new feature, or component, in its context.</em>

## How to test it?
<em>Describe the steps with bullet points. Is there any external reference, link, or example?</em> 

## References
<em>Spread the knowledge: is this any content you used to create this PR that is worth sharing?</em>  

<em>Extra tip: add references to related issues or mention people important to this PR may be good for the documentation and reviewing process</em>

### CODE REVIEW

File: src/custom-sw-code.js [hunk 1]
```diff
@@ -24,11 +24,9 @@ const catchHandler = async ({ event }) => {
 
   if (dest === 'document') {
```
Comment (lariciamota) [inline src/custom-sw-code.js:25]: Just for me to understand, why were these changes necessary in this file?

File: src/components/sections/ImageGallery/ImageGallery.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      image={image}
+      alt={props.alt}
+    />
+  ))
+
+  return (
+    <>
+      <div className={clsx(styles.container, styles.onlyDesktop)}>{images}</div>
+      <div className={clsx(styles.container, styles.onlyMobile)}>
+        <Carousel>{images}</Carousel>
```
Comment (lariciamota) [inline src/components/sections/ImageGallery/ImageGallery.tsx]: The pagination visual representation is going to be added in another PR?
Screen Shot 2021-08-20 at 12 52 14

File: src/custom-sw-code.js (hunk 1 above)
Comment (emersonlaurentino) [reply src/custom-sw-code.js:25]: sonarqube problem

File: src/components/sections/ImageGallery/ImageGallery.tsx (hunk 2 above)
Comment (emersonlaurentino) [reply src/components/sections/ImageGallery/ImageGallery.tsx]: in another PR, because it's the same pagination on the carousel at home.
